software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -124,11 +125,21 @@ public void install() {
     @Override
     public void customize() {
         Networking.checkPortsValid(MutableMap.of("amqpPort", getAmqpPort()));
-        newScript(CUSTOMIZING)
-                .body.append(
-                    format("cp -R %s/* .", getExpandedInstallDir())
-                )
-                .execute();
+        ScriptHelper scriptHelper = newScript(CUSTOMIZING);
+
+        scriptHelper.body.append(
+                format("cp -R %s/* .", getExpandedInstallDir())
+        );
+
+        if (Boolean.TRUE.equals(entity.getConfig(RabbitBroker.ENABLE_MANAGEMENT_PLUGIN))) {
+            scriptHelper.body.append(
+                    "./sbin/rabbitmq-plugins enable rabbitmq_management"
+            );
+        }
+        scriptHelper.failOnNonZeroResultCode();
+        scriptHelper.execute();
+
+        copyTemplate(entity.getConfig(RabbitBroker.CONFIG_TEMPLATE_URL), getConfigPath() + ".config");
```

Comment:
It's intentional - I don't know why but rabbitmq adds the .config to the RABBITMQ_CONFIG_FILE env variable